client.go

```diff
@@ -285,6 +290,32 @@ func (client *Client) GetNetwork() map[string]AccountID {
 	return client.network.network
 }
 
+func (client *Client) SetMaxBackoff(max time.Duration) {
```

Comment:
This should return `*Client` much like the other set methods.

Reply:
Most others don't though